src/modules/0.0.22/aws_s3/rpcs/s3_clean_bucket.ts

```diff
@@ -3,6 +3,7 @@ import { S3, _Object } from '@aws-sdk/client-s3';
 import { AwsS3Module } from '..';
 import { AWS, crudBuilder2, crudBuilderFormat } from '../../../../services/aws_macros';
 import { Context, RpcBase, RpcResponseObject } from '../../../interfaces';
+import { BucketObject } from '../entity';
```

Comment:
Just a comment: We haven't release v0.0.22 yet, which is also the first iteration that has this RPC function. With this PR the "normal" cleaning logic should work because deleting all of the bucket object records should trigger a delete on all of those files, so we can probably just drop this RPC function entirely?

Reply:
maybe we can do in a follow up ? right now it is used on the ci for prisma, for example, and in codepipeline as well. I prefer if we do in separate prs, so we also refactor the dependencies